src/Model/Path/Output.php

```diff
@@ -6,8 +6,14 @@
 
 interface Output
 {
+    public const CONTENT_TYPE_APPLICATION_JSON = 'application/json';
+
+    public const CONTENT_TYPE_APPLICATION_PROBLEM_JSON = 'application/problem+json';
+
     /**
      * @return mixed
      */
     public function example();
+
+    public function contentType(): string;
```

Comment:
Should we maybe allow to return a collection of content types ? The same output could be represented in different formats like JSON or XML.

For now we could return just JSON in the actual output implementations

Reply:
We could but I am not sure if it's worth changing now, because then we should allow passing content types to at least one of the `Output` implementations, otherwise it would be a dead functionality.

Reply:
True, we need to allow this to be configured in the output, but the interface should maybe allow for it now. It just happens that the implementation will now support configuration. I am thinking of doing this for the interface now just to keep BC limited in the future

Reply:
I've added multiple content types to the interface